controllers/comment.controller.js

```diff
@@ -14,4 +14,27 @@ exports.getCommentsByArticleId = (req, res, next) => {
 			res.status(200).send({ comments });
 		})
 		.catch(next)
-}
\ No newline at end of file
+}
+exports.postCommentByArticleId = (req, res, next) => {
+	const { article_id } = req.params;
+	const { username, body } = req.body;
+
+	if (!username || !body) {
+		return res.status(400).send({ message: 'Bad Request' })
+	}
+
+	selectArticleById(article_id)
+		.then((article) => {
+			if (!article) {
+				req.status(404).send({ message: err.message });
+			} else {
+				insertComment(article_id, username, body)
+					.then((comment) => {
+						res.status(201).send({ comment })
+					})
+					.catch(next);
+			}
+		})
+		.catch(next)
```

Comment:
You'll only ever need 1 catch block for 1 function 👀

Reply:
done